test/powershell/Host/HostUtilities.Tests.ps1

```diff
@@ -0,0 +1,68 @@
+﻿Describe "InvokeOnRunspace method argument error handling" -tags "CI" {
+
+    $command = [System.Management.Automation.PSCommand]::new()
+    $localRunspace = $host.Runspace
+
+    $ex = $null
+    try
```

Comment:
The test code should be within the `It` block.

Also, these tests don't follow our guidelines for tests that expect exceptions, see [https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md)

Note that the exception thrown on failure should have a better message than in the guidelines, e.g.

```powershell
    throw "Command did not throw exception"
```